benchmarks/link-to/active/bench.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
we could likely add "what versions this tests works for" stuff to package.json. For now i'll just add a disabled falg